apps/token/src/routes/governance/propose/propose.tsx

```diff
@@ -79,6 +79,16 @@ export const Propose = () => {
               </Link>
             </p>
           </li>
+          <li>
+            <p>
+              <Link
+                className="underline"
+                to={`${Routes.GOVERNANCE}/propose/update-asset`}
```

Comment:
thought (non-blocking): we store the top level routes in a constant I wonder if we want to add a ticket to store child routes names in an object as well to cu down on magic strings. Or maybe we should utilise route names more and just store those in an object.

Reply:
https://github.com/vegaprotocol/frontend-monorepo/issues/1607